lib/resource-mapper.js

```diff
@@ -149,6 +147,25 @@ class ResourceMapper {
     return { path, contentType: contentType || this._defaultContentType }
   }
 
+  // encode/decode path except slash/%encodedSlash/double%encodedSlash
+  _exceptSlash () { return /(\/|%2f|%2F|%252F|%252f)/g }
```

Comment:
Good find about the `%25`. However, we found that we also need to guard against _sequences_ of `%25252525` in https://github.com/CommunitySolidServer/CommunitySolidServer/pull/1273.